### PR TITLE
Fixed Typographical Error in LICENSE

### PR DESCRIPTION
**Pull request by iamrajiv**
#### Brief description of what is fixed or changed

Fixed Typographical Error in LICENSE

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by iamrajiv** [comment]
Thankyou @smichr for the suggestion you have given me  in PR #15734 I have implemented the suggestion in this PR  please review this code

File: LICENSE [hunk 1]
```diff
@@ -64,7 +64,7 @@ DAMAGE.
 
 --------------------------------------------------------------------------------
 
-Submodule taken from the multipledispatch project (https://github.com/mrocklin/multipledispatch)
+Submodule that were taken from the multiple dispatch project (https://github.com/mrocklin/multipledispatch)
```
Comment (smichr) [inline LICENSE]: `Submodules taken from the multiple-dispatch project...`

File: LICENSE [hunk 2]
```diff
@@ -79,7 +79,7 @@ modification, are permitted provided that the following conditions are met:
   b. Redistributions in binary form must reproduce the above copyright
      notice, this list of conditions and the following disclaimer in the
      documentation and/or other materials provided with the distribution.
-  c. Neither the name of multipledispatch nor the names of its contributors
+  c. Neither the name of multiple dispatches nor the names of its contributors
```
Comment (smichr) [inline LICENSE]: this is the name of the project and it shouldn't be changed

Comment (iamrajiv) [reply LICENSE]: got it thanks for suggestion @smichr

File: LICENSE (hunk 1 above)
Comment (iamrajiv) [reply LICENSE]: thanks for suggestion @smichr

**Pull request merged**